test/lib/numpy

```diff
@@ -1 +1 @@
-Subproject commit e4d4b45d26813899c66e112365896aabbb7b19fa
+Subproject commit f92576298546ffe984a2df466a80bb6a1d2e60c9
```

Comment:
I think it's nicer to update to the last released version and not some unreleased random revision.. maybe we should take 4092a9e160cc247a4a45724579a0c829733688ca which is https://github.com/numpy/numpy/commits/v1.11.0